data/language/dutch.txt

```diff
@@ -1693,8 +1693,8 @@ STR_1691    :{WINDOW_COLOUR_2}  Kosten: {BLACK}{CURRENCY}
 STR_1692    :{WINDOW_COLOUR_2}  Kosten: {BLACK}vanaf {CURRENCY}
 STR_1693    :{SMALLFONT}{BLACK}Bezoekers
 STR_1694    :{SMALLFONT}{BLACK}Werknemers
-STR_1695    :{SMALLFONT}{BLACK}Inkomsten en kosten
-STR_1696    :{SMALLFONT}{BLACK}Klanteninformatie
+STR_1695    :{SMALLFONT}{BLACK}Inkomsten en uitgaven
+STR_1696    :{SMALLFONT}{BLACK}Klantinformatie
```

Comment:
Dit is wel correct. Het is een vaste verbinding en het gaat over één database vol informatie. Al kan het in dit spel ook over klantinformatie gaan waarin daadwerkelijk maar 1 klant staat, als je net een nieuw park begint.

In het Engels gedragen customers/klanten en customer information/klantinformatie zich hetzelfde.